documentation.html

```diff
@@ -768,8 +771,281 @@ <h6 class="text-regular-weight product-title">
+                    >Enter Your Birthdate</label
+                  >
+                </div>
+                <div class="column-80-pc">
+                  <input class="form-field" type="date" required />
+                  <div class="form-validation-msg form-field-error">
+                    <span class="form-field-symbol"
+                      ><i class="fas fa-exclamation-circle"></i></span
+                    >Please fill birthdate!
+                  </div>
+                  <div class="form-validation-msg form-field-success">
+                    <span class="form-field-symbol"
+                      ><i class="fas fa-check-circle"></i></span
+                    >check pass
+                  </div>
+                </div>
+              </div>
+
+              <div class="row">
+                <div>
+                  <input class="form-checkbox-field" type="checkbox" required />
+                  <label class="form-label-required-field">
+                    Yes, I accept the terms.</label
+                  >
+                  <div class="form-validation-msg form-field-error">
+                    <span class="form-field-symbol"
+                      ><i class="fas fa-exclamation-circle"></i></span
+                    >Please check the box!
+                  </div>
+                  <div class="form-validation-msg form-field-success">
+                    <span class="form-field-symbol"
+                      ><i class="fas fa-check-circle"></i></span
+                    >check pass
+                  </div>
+                </div>
+              </div>
+
+              <button class="btn btn-outline-primary btn-sm-size" type="submit">
+                Submit
+              </button>
+            </form>
+          </div>
+          <div class="code-snippet-display">
+            <iframe
+              src="https://www.thiscodeworks.com/embed/620ea4cc14fadf0015661d33"
+              style="width: 100%; height: 1113px"
+              frameborder="0"
+            ></iframe>
+          </div>
+        </div>
+        <hr />
+        <div id="Image">
+          <div class="component-intro">
+            <h1>Images</h1>
+            <p class="body-cp-lg text-light-weight">
+              Images can be responsive to fit the parent's width, and also can
+              be customised to be round shaped
+            </p>
+          </div>
+
+          <div class="component-display-container display-flex-column">
+            <img
+              loading="lazy"
+              class="img-responsive"
+              src="./Components/images/undraw_image_post_re_25wd.svg"
+            />
+          </div>
+          <div class="code-snippet-display">
+            <iframe
+              src="https://www.thiscodeworks.com/embed/620ea61014fadf0015661d34"
+              style="width: 100%; height: 125px"
+              frameborder="0"
+            ></iframe>
+          </div>
+
+          <h2 class="h4">Example of Round Image</h2>
```

Comment:
Using h2 tag and adding h4 class looks a little off. Instead of h4 class name you can give a different name to it. Avoid using class names same as available html elements.